prerequisites/build-functions/build_and_install.sh

```diff
@@ -23,8 +23,8 @@ build_and_install()
 
     if [[ "${package_to_build}" == "mpich" && "${version_to_build}" == "3.2" ]]; then
       info "Patching MPICH 3.2 on Mac OS due to segfault bug (see http://lists.mpich.org/pipermail/discuss/2016-May/004764.html)."
-      sed -i '' 's/} MPID_Request ATTRIBUTE((__aligned__(32)));/} ATTRIBUTE((__aligned__(32))) MPID_Request;/g' \
-	  "${download_path}/${package_source_directory}/src/include/mpiimpl.h"
+      sed 's/} MPID_Request ATTRIBUTE((__aligned__(32)));/} ATTRIBUTE((__aligned__(32))) MPID_Request;/g' \
+        "${download_path}/${package_source_directory}/src/include/mpiimpl.h"
```

Comment:
ah I'm guessing this is due to the non-posix compliant `-i` flag being weird on different systems.... I think maybe we should just use two lines: redirect sed (without `-i`) into a `file.new` and then `cp file.new file`. We could potentially use pure bash and `while read line ; do ... done < file`  but I think this is excessive. I'll fix this.